src/main/java/model/janggi_board/JanggiBoard.java

```diff
@@ -0,0 +1,110 @@
+package model.janggi_board;
+
+import static model.janggi_board.JanggiBoardSetUp.DEFAULT_SETUP;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+import model.Path;
+import model.Point;
+import model.Team;
+import model.piece.Piece;
+
+public class JanggiBoard {
+    public static final int VERTICAL_SIZE = 10;
+    public static final int HORIZONTAL_SIZE = 9;
+    private ArrayList<ArrayList<Dot>> janggiBoard;
```

Comment:
```suggestion
    private List<List<Dot>> janggiBoard;
```
장기보드를 위와 같이 만들어주는건 어떨까요?
SOLID 와도 연관이 있는 부분이랍니다 :)

(만약 적용하신다면, janggiBoard 를 만드는 로직에도 적용해주세요)

Reply:
지금 생각하니 List<List<>>인 편이 더 구현 List 종류 변경에 유연하게 대응할 수 있겠네요! 페어도 한번 제안한 부분이었는데, 그냥 앞 부분에서 확실히 타입을 명시하는 게 좋지 않겠냐 하고 밀어붙였던 것 같습니다.. 수정하겠습니다!